indico/web/client/js/react/components/pictures/PictureArea.jsx

```diff
@@ -5,6 +5,7 @@
 // modify it under the terms of the MIT License; see the
 // LICENSE file for more details.
 
+import omit from 'lodash/omit';
```

Comment:
IIRC these single-function lodash imports are deprecated.

So I'd rather do the usual `import _ from 'lodash';` and then use `_.omit(...)`